inlang/source-code/sdk/src/v2/mocks/multipleMatcher/bundle.ts

```diff
@@ -0,0 +1,188 @@
+import type { MessageBundle } from "../../types.js"
+
+export const multipleMatcherBundle: MessageBundle = {
+	id: "mock-bundle-human-id",
+	alias: {
+		default: "mock-bundle-alias",
+	},
+	messages: [
+		{
+			locale: "en",
+			declarations: [
+				{
+					type: "input",
+					name: "username",
+					value: {
+						type: "expression",
+						arg: {
+							type: "variable",
+							name: "username",
+						},
+					},
+				},
+			],
+			selectors: [
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "count: plural",
```

Comment:
The plural function is an annotation to the variable expression
```js
{
  type : "expression",
  arg: { type : "variable", name: "count"},
  annotation: { type: "function", name: "plural", options: [] }
}
```

Reply:
from here
https://github.com/opral/monorepo/pull/2795#pullrequestreview-2062588832

Reply:
Done